Document how to delete all sessions for both file and redis storage (#4018)
tracim/tracim: backend/doc/setting.md

regularly (for example by using a cron job), which will delete file which have not been modified since 10 days.
You should use this command in both session data and session lock dirs.

> ## delete all existing sessions

Author: ```suggestion
#### delete all existing sessions (file storage)
```

regularly (for example by using a cron job), which will delete file which have not been modified since 10 days.
You should use this command in both session data and session lock dirs.

## delete all existing sessions

```shell

> # note: <session.data_dir> refer to absolute path given by config parameter session.data_dir

Reviewer: ```suggestion
# note: <session.data_dir> refers to the absolute path given by the config parameter `session.data_dir`.
```

    session.type = ext:redis
    session.url = redis://localhost:6379/0

> ## delete the existing sessions

Author: ```suggestion
#### delete the existing sessions (redis storage)
```

    session.type = ext:redis
    session.url = redis://localhost:6379/0

> ## delete all existing sessions

Reviewer: ```suggestion
## delete the existing sessions
```

```shell

> # note: <session.url> refer to value of config parameter session.url (redis url)

Reviewer: ```suggestion
# note: <session.url> refers to the value of the config parameter `session.url` (redis url)
```
